packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -122,13 +154,12 @@ class SlidePanel extends React.Component {
     this.panelNode = node;
   }
 
-  setLastClicked(event) {
-    this.lastClicked = event.target;
-  }
-
   setDisclosingNode(node) {
     if (node) {
-      node.setAttribute('aria-expanded', 'true');
+      if (roleIsCompartible(node)) {
+        // Set aria-expanded attribute to true for nodes with associated roles or no role only
+        node.setAttribute('aria-expanded', 'true');
+      }
```

Comment:
This node is not being expanded.  I don't think we should be adding this role.

Reply:
aria-expanded attribute wasn't added by me, it was there before the change, I just added a role check, as it can be added [only with specific roles](aria-expanded attribute), otherwise it causes accessibility violation. I'll double check with @mjpalazzo if aria-expanded attribute needs to be removed.